### ISSUE TITLE
Issue in using export default and evalCode

### ISSUE THREAD

**Issue by dod91** [justjake/quickjs-emscripten#79]
Hello, first of all thank you for the great work on this library.

I'm not sure if I'm missing something but when running `vm.evalCode` with the following code:
```
function testFn(){
  console.log('Test');
}

export default testFn;
```
I get this error:
```
{
  name: 'SyntaxError',
  message: 'unsupported keyword: export'
}
```

While running this, it works as expected:
```
export default function testFn(){
  console.log('Test');
}
```

I've also tried by specifying as `options`:
```
{
  type: 'module',
  compileOnly: true
}
```

Not sure if I'm missing something in the usage or in code. 
Here's a small script to reproduce the error:
```
import { getQuickJS, Scope } from 'quickjs-emscripten';

const notWorkingCode = `
function testFn(){
  console.log('Test');
}

export default testFn;`;

const workingCode = `
export default function testFn(){
  console.log('Test');
}`;

async function main() {
  const QuickJS = await getQuickJS();
  return Scope.withScope((scope) => {
    const vm = scope.manage(QuickJS.newContext());
    const options = {
      type: 'module',
      compileOnly: true,
    };
    const result = vm.evalCode(notWorkingCode, options);
    if (result.error) {
      const dump = result.error.consume((error) => vm.dump(error));
      console.log(dump);
      return;
    }
    console.log('All good');
  });
}

main()
```

**Comment by dod91** [comment]
An additional note, the following code works as expected:
```
export default testFn;

function testFn(){
  console.log('Test');
}
```

**Comment by dod91** [comment]
Following up on this as I think I found the root cause of it.
In code above the `evalCode` function is called like this:
```
const result = vm.evalCode(notWorkingCode, options); 
```

After some investigation I found that the `evalCode` method in the `QuickJSContext` class exposes the options as third parameter (code reference [here](https://github.com/justjake/quickjs-emscripten/blob/main/ts/context.ts#L671)). I'm not sure if this is documented.

So the right way of calling the `evalCode` method from a QuickJSContext passing options is the following:

```
const result = vm.evalCode(notWorkingCode, 'eval.js', options); 
```
